Formula/r/rsync.rb

```diff
@@ -33,15 +33,6 @@ class Rsync < Formula
 
   uses_from_macos "zlib"
 
-  # hfs-compression.diff has been marked by upstream as broken since 3.1.3
-  # and has not been reported fixed as of 3.2.7
-  patch do
-    url "https://download.samba.org/pub/rsync/src/rsync-patches-3.3.0.tar.gz"
-    mirror "https://www.mirrorservice.org/sites/rsync.samba.org/rsync-patches-3.3.0.tar.gz"
-    sha256 "3dd51cd88d25133681106f68622ebedbf191ab25a21ea336ba409136591864b0"
-    apply "patches/fileflags.diff"
```

Comment:
Any reason to stop applying `patches/fileflags.diff`? Took a peek at the changelog but wondering if I missed something.

Reply:
yeah, we should drop it, I will add it back (unclear how these patches would be merged into main repo)

---

fwiw, this is the main repo, https://github.com/RsyncProject/rsync and this is the patch repo, https://github.com/RsyncProject/rsync-patches

Reply:
No worries, I was just curious, since I know that patch has been included in the formula for a while now, so I was just wondering if there was some reason to drop it on this release in particular.